packages/driver/src/cy/chai/inspect.ts

```diff
@@ -54,6 +54,18 @@ export function create (chai) {
       typeof object.nodeName === 'string'
   }
 
+  // We can't just check if object instanceof ShadowRoot, because it might be the document of an iframe,
+  // which in Chrome 99+ is a separate class, and instanceof ShadowRoot returns false.
+  const isShadowRoot = function (object) {
```

Comment:
Does this change with chrome has impact on the implementation of the `cy.shadow()` command?